engine/apps/alerts/tasks/check_escalation_finished.py

```diff
@@ -96,38 +95,27 @@ def audit_alert_group_escalation(alert_group: "AlertGroup") -> None:
     task_logger.info(f"{base_msg} passed the audit checks")
 
 
-def get_auditable_alert_groups_started_at_range() -> typing.Tuple[datetime.datetime, datetime.datetime]:
-    """
-    NOTE: this started_at__range is a bit of a hack..
-    we wanted to avoid performing a migration on the alerts_alertgroup table to update
-    alert groups where raw_escalation_snapshot was None. raw_escalation_snapshot being None is a legitimate case,
-    where the alert group's integration does not have an escalation chain associated with it.
-
-    However, we wanted a way to be able to differentiate between "actually None" and "there was an error writing to
-    raw_escalation_snapshot" (as this is performed async by a celery task).
-
-    This field was updated, in the commit that added this comment, to no longer be set to None by default.
-    As part of this celery task we do a check that this field is in fact not None, so if we were to check older
-    alert groups, whose integration did not have an escalation chain at the time the alert group was created
-    we would raise errors
-    """
-    return (datetime.datetime(2023, 3, 25, tzinfo=pytz.UTC), timezone.now() - datetime.timedelta(days=2))
```

Comment:
this is no longer relevant and is safe to remove. We are only looking at alert groups for the past 2 days. It has been ~4 months since the release where `raw_escalation_snapshot` is no longer being set to `None`.